### ISSUE TITLE
Clear single refinements

### ISSUE THREAD

**Issue by MrMooky** [algolia/instantsearch.js#3922]
Something similar has been asked before, but with a different output wanted.

What we basically need is a way to clear these 9 filters separately and **not** all at once with the current [clearRefinements](https://www.algolia.com/doc/api-reference/widgets/clear-refinements/js/).

<img width="235" alt="Screen Shot 2019-07-05 at 14 51 39" src="https://user-images.githubusercontent.com/3603346/60723612-6863d680-9f34-11e9-8239-b35438270b3d.png">

So basically `clearRefinements` for every single "filtergroup". Is something like this possible with instantsearch.js?

**Comment by Haroenv** [comment]
This is possible if you use the `transformItems` to filter out the  refinements & only keep the one you actually want to clear. Hope that helps!

**Comment by tkrugg** [comment]
Hi @MrMooky another option could be to leverage `includeAttributes`.
Here is an example that shows how it's done
https://codesandbox.io/s/ais-ecommerce-demo-app-yxvf8

I hope this helps. I'll close this for now. Do not hesitate to reopen if this hasn't solved your issue!

**Comment by MrMooky** [comment]
@tkrugg That solves my issue. Thanks a lot. Easier than I though. :)
